crates/pg_cli/src/cli_options.rs

```diff
@@ -18,6 +18,10 @@ pub struct CliOptions {
     #[bpaf(long("use-server"), switch, fallback(false))]
     pub use_server: bool,
 
+    /// Skip over files containing syntax errors instead of emitting an error diagnostic.
```

Comment:
Maybe I am misunderstanding, but is this the correct comment? I thought it would simply not connect to the db and still provide everything we can like lint and Syntax diagnostics

Reply:
no, sorry, that's just a copy-paste error! good catch.